src/ceph-volume/ceph_volume/devices/raw/prepare.py

```diff
@@ -64,21 +94,20 @@ def safe_prepare(self, args=None):
             logger.info('will rollback OSD ID creation')
             rollback_osd(self.args, self.osd_id)
             raise
-        terminal.success("ceph-volume raw prepare successful for: %s" % self.args.data)
-
-    def get_cluster_fsid(self):
-        """
-        Allows using --cluster-fsid as an argument, but can fallback to reading
-        from ceph.conf if that is unset (the default behavior).
-        """
-        if self.args.cluster_fsid:
-            return self.args.cluster_fsid
```

Comment:
Is this code meant to be removed? The option is still offered by the cli (see https://github.com/ceph/ceph/blob/79104332d98d5857fc74285320102fc232e6e0ed/src/ceph-volume/ceph_volume/devices/raw/common.py#L29)?
Seems like it was never used, so maybe also drop the cli arg?